bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -767,11 +806,10 @@ bool ReplicaImp::validatePreProcessedResults(const PrePrepareMsg *msg) {
   errors.resize(error_id);
   for (auto err : errors) {
     if (err) {
-      // trigger view change
-      LOG_ERROR(VC_LOG,
-                "Received PrePrepareMsg containing PreProcessResult with invalid signature: "
-                    << *err << ". Ask to leave view " << getCurrentView());
-      askToLeaveView(ReplicaAsksToLeaveViewMsg::Reason::PrimarySentBadPreProcessResult);
+      // Indicate a view change
+      LOG_WARN(VC_LOG,
+               "Received PrePrepareMsg containing PreProcessResult with invalid signature: "
```

Comment:
If touched, please use KVLOG.

Reply:
Thanks @yuliasherman for your comment, I have incorporated the changes.